diff --git a/base/BUILD.gn b/base/BUILD.gn
index d50dfa6f77a89c60ca27598876a9a2920d65cf11..3d765137551e786e6c6f6b512d531eff09bd1bbb 100644
--- a/base/BUILD.gn
+++ b/base/BUILD.gn
@@ -4378,6 +4378,7 @@ if (is_android) {
       "android/java/src/org/chromium/base/task/ThreadPoolTaskExecutor.java",
       "android/java/src/org/chromium/base/task/UiThreadTaskExecutor.java",
     ]
+    sources += brave_java_base_sources
 
     if (!is_cronet_build) {
       sources += [ "android/java/src/org/chromium/base/IntentUtils.java" ]
