diff --git a/chrome/browser/about_flags.cc b/chrome/browser/about_flags.cc
index 628b87d2c578285c47c69255eba0c3cc1523e524..b0d26376e2bf3b96e3489ab54d758299054a897b 100644
--- a/chrome/browser/about_flags.cc
+++ b/chrome/browser/about_flags.cc
@@ -3781,6 +3781,7 @@ const FeatureEntry::FeatureVariation kPasswordGenerationExperimentVariations[] =
 //
 // When adding a new choice, add it to the end of the list.
 const FeatureEntry kFeatureEntries[] = {
+    BRAVE_ABOUT_FLAGS_FEATURE_ENTRIES
 // Include generated flags for flag unexpiry; see //docs/flag_expiry.md and
 // //tools/flags/generate_unexpire_flags.py.
 #include "build/chromeos_buildflags.h"
